pkg/operator/ceph/cluster/mgr/spec_test.go

```diff
@@ -69,7 +71,7 @@ func TestPodSpec(t *testing.T) {
 
 	podTemplate := cephtest.NewPodTemplateSpecTester(t, &d.Spec.Template)
 	podTemplate.RunFullSuite(config.MgrType, "a", appName, "ns", "ceph/ceph:myceph",
-		"100", "1337" /* resources */)
```

Comment:
What I mean is that both the limits and requests in all the tests use the exact same values, "100", and "1337". I think that we should either eliminate the redundant `, "100", "1337"` from the function calls and function definition, or we should alter the tests so that these values are different. Otherwise, we are wasting line space with repetition.